### ISSUE TITLE
error while visiting tag detail page

### ISSUE THREAD

**Issue by developer-guy** [klausmeyer/docker-registry-browser#247]
```
TypeError (no implicit conversion of Hash into String):
app/models/tag.rb:22:in `find'
app/controllers/tags_controller.rb:21:in `find_tag'
```

**Comment by klausmeyer** [comment]
Could you please provide more information about this issue?
Otherwise it's nearly impossible to debug it.

* What version of the application are you running?
* Please provide the full stack-trace / error output of the failing request.

**Comment by codescale** [comment]
Same here with version 1.3.0

There are the following lines in the log
```
D, [2019-12-03T15:16:30.884545 #1] DEBUG -- response: age: "1441"
date: "Tue, 03 Dec 2019 15:16:43 GMT"
connection: "Keep-Alive"
via: "NS-CACHE-10.0:  61"
server: "Nexus/3.16.1-02 (OSS)"
x-content-type-options: "nosniff"
content-security-policy: "sandbox allow-forms allow-modals allow-popups allow-presentation allow-scripts allow-top-navigation"
x-xss-protection: "1; mode=block"
last-modified: "Thu, 07 Nov 2019 07:33:02 GMT"
docker-content-digest: "sha256:3eb9abd560a2915faa7aeecae9beb359f9ce9b8cfedfbdc8fb45ffc4a4ac2209"
docker-distribution-api-version: "registry/2.0"
content-type: "application/vnd.docker.container.image.v1+json"
content-length: "7520"
strict-transport-security: "max-age=157680000"
I, [2019-12-03T15:16:30.884961 #1]  INFO -- : [c706d8e5-4aba-4ead-a0ca-8541671ce373] Completed 500 Internal Server Error in 336ms (Allocations: 4022)
F, [2019-12-03T15:16:30.885330 #1] FATAL -- : [c706d8e5-4aba-4ead-a0ca-8541671ce373]   
[c706d8e5-4aba-4ead-a0ca-8541671ce373] TypeError (no implicit conversion of Hash into String):
[c706d8e5-4aba-4ead-a0ca-8541671ce373]   
[c706d8e5-4aba-4ead-a0ca-8541671ce373] app/models/tag.rb:22:in `find'
[c706d8e5-4aba-4ead-a0ca-8541671ce373] app/controllers/tags_controller.rb:21:in `find_tag'
```

Does this help?

**Comment by klausmeyer** [comment]
> Does this help?

@codescale I think this will help for sure. From the output I can already see that you're using a different registry implementation (Nexus) which seems to behave differently than the official docker registry.



> And after I switched to 1.2.3 the error disappeared.

This is also very helpful because then I think it might be related with a feature I've recently implemented in 1.3.0.

Will look into it.

**Comment by klausmeyer** [comment]
Think I was able to reproduce and fix the issue.
Please pull `latest` tag and verify in our installation. I'll issue a new release if it is working.

**Comment by klausmeyer** [comment]
@codescale / @developer-guy ping 😁

**Comment by codescale** [comment]
Hi, I can verify this next Monday. Sorry for the delay. (and thx for the support :) )

**Comment by klausmeyer** [comment]
Perfect. Thank you 🌟

**Comment by codescale** [comment]
Awesome. It works with `latest` tag (And says version `1.3.0 (169e212)` in footer)! Thx.

**Comment by klausmeyer** [comment]
Thanks for testing it. I've just released version `1.3.1` which will be available soon.
